lib/jsrt/JsrtCommonExports.inc

```diff
@@ -11,6 +11,7 @@
     JsGetContextData
     JsSetContextData
     JsRunScript
+    JsExperimentalRunModule
```

Comment:
I kinda want this to be named `JsExperimentalApiRunModule` so that the self reference is clear and unambiguous.  What do you think?